test/e2e/collector/eraser_test.go

```diff
@@ -0,0 +1,116 @@
+//go:build collector
+// +build collector
+
+package collector
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	eraserv1alpha1 "github.com/Azure/eraser/api/v1alpha1"
+	"github.com/Azure/eraser/test/e2e/util"
+	// appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	// clientgo "k8s.io/client-go/kubernetes"
+
+	"sigs.k8s.io/e2e-framework/klient/wait"
+	"sigs.k8s.io/e2e-framework/klient/wait/conditions"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+func TestRemoveImagesFromAllNodes(t *testing.T) {
+	const (
+		alpine = "alpine"
+	)
+
+	collectScanErasePipelineFeat := features.New("Test Remove Image From All Nodes").
+		Assess("ImageCollector CR is generated", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			c, err := cfg.NewClient()
+			if err != nil {
+				t.Error("Failed to create new client", err)
+			}
+
+			resource := eraserv1alpha1.ImageCollector{}
+			wait.For(func() (bool, error) {
+				err := c.Resources().Get(ctx, "imagecollector-shared", "default", &resource)
```

Comment:
What's this CR used for?

Reply:
It's the aggregated collector list, produced by combining the collector lists from the individual nodes.